publish.sh

```diff
@@ -22,6 +22,10 @@ for i in ${CLUSTERS}; do
     docker tag ${TARGET_REPO}/k8s-$i ${TARGET_REPO}/k8s-$i:${KUBEVIRTCI_TAG}
 done
 
+# Provision alpine container disk and tag it
+(cd cluster-provision/images/vm-image-builder && ./create-containerdisk.sh alpine-cloud-init)
+docker tag  alpine-cloud-init:devel ${TARGET_REPO}/alpine-container-disk:${KUBEVIRTCI_TAG}
```

Comment:
nit: make the repo name the same as the image

```suggestion
docker tag  alpine-cloud-init:devel ${TARGET_REPO}/alpine-cloud-init-container-disk:${KUBEVIRTCI_TAG}
```

Reply:
I am going to rename this similart to fedora "with-testing-tools" and at follow up PR rename the the original image too.